LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1790,6 +1790,9 @@ webkit.org/b/259160 http/tests/privateClickMeasurement/attribution-conversion-th
 webkit.org/b/259160 http/tests/privateClickMeasurement/triggering-event-with-attribution-source-through-fetch-keepalive.html [ Pass Failure ]
 webkit.org/b/259160 http/tests/privateClickMeasurement/triggering-event-with-attribution-source-through-fetch-keepalive-ephemeral.html [ Pass Failure ]
 
+webkit.org/b/261118 [ x86_64 ] imported/w3c/web-platform-tests/webcodecs/temporal-svc-encoding.https.any.worker.html?h264 [ Pass Failure ]
+webkit.org/b/261118 [ x86_64 ] imported/w3c/web-platform-tests/webcodecs/temporal-svc-encoding.https.any.html?h264 [ Pass Failure ]
```

Comment:
I don't think this should reference this bug, which will get closed, should be a new one.

Reply:
Filed https://bugs.webkit.org/show_bug.cgi?id=263112